.golangci.yml

```diff
@@ -0,0 +1,71 @@
+run:
+  # Timeout for analysis, e.g. 30s, 5m, default is 1m
+  deadline: 3m
```

Comment:
do we need to increase the default?

Reply:
1m sounds too short. I already see errors after 28 seconds.

Reply:
yes, makes sense. Just something to keep in mind and maybe review later